automation_tools/satellite6/hammer.py

```diff
@@ -399,5 +399,8 @@ def hammer_determine_cv_and_env_from_ak(ak_name, organization_id):
     """
     data = hammer('activation-key info --name {0} --organization-id '
                   '{1}'.format(ak_name, organization_id))
+    if not isinstance(data, list) and not isinstance(data, dict):
```

Comment:
This condition can be simplified as `if not isinstance(data, (dict, list)):`

Simple testing:

```python
In [2]: isinstance([], (list, dict))
Out[2]: True

In [3]: isinstance({}, (list, dict))
Out[3]: True
```